mubench.pipeline/tasks/configurations/detector_interface_configuration.py

```diff
@@ -0,0 +1,11 @@
+RUN_FILE = "run.yml"
+FINDINGS_FILE = "findings.yml"
```

Comment:
The above two lines are unnecessary, because this data is in `DetectorRun` now, right?

Can we move the other constants closer to the detector interfaces?